docs/src/modules/components/MarkdownElement.js

```diff
@@ -322,10 +352,13 @@ const Root = styled('div')(
       '&.MuiCallout-warning': {
         color: `var(--muidocs-palette-grey-900, ${lightTheme.palette.grey[900]})`,
```

Comment:
I always wondered if this was a mistake or not (to not use the warning palette).

```suggestion
        color: `var(--muidocs-palette-warning-900, ${lightTheme.palette.warning[900]})`,
```

But on the other hand, the warning variant feels great in grey, a bit better than the other alert variants that are colored. It's pretty much a question, of what's best? https://mintlify.com/docs/content/components/callouts#warning-callouts or https://nextra.site/docs/guide/built-ins/callout#example.

Reply:
Mostly because of contrast, I think ⎯ the darkest warning color seems too tinted in my perspective, so I thought that using something closer to "true" black made more sense. But, nice links! I was looking for the Mintlify examples yesterday and couldn't find it ⎯ and there it goes 😬